### DISCUSSION TITLE
Ram Timings for version 1.7.0

### DISCUSSION THREAD

**Discussion by hanai3Bi** [hanai3Bi/Switch-OC-Suite#30] [Ram Timing | Customization]
[Mariko]
Samsung K4U6E3S4AA-MGCL
AUTO_ADJ_ALL
2502Mhz
VDD2: 1175, VDDQ: 600
DVB Table Shift: 1
5-4-4-4-5-5-4
edit: changed to 4-3-4-4-3-5-3 for stability with OW2

GPU volt(speedo 1625) : 640,640,640,640,640,640,640,640,640,640,640,640,675,710,735,,770, 800

**Comment by Git7234217** [comment]
Samsung K4U6E3S4AA-MGCL

CUSTOM_ADJ_ALL
UV2
2480mhz
620-1200
DVB Table Shift:1
4544554
![2023070615133000-DB1426D1DFD034027CECDE9C2DD914B8](https://github.com/hanai3Bi/Switch-OC-Suite/assets/138766339/d94726a6-0d45-4ed2-b02b-745d43606829)

**Comment by halop** [comment]
[Mariko]
Samsung K4U6E3S4AA-MGCL
AUTO_ADJ_ALL
2560Mhz
VDD2: 1200, VDDQ: 630
DVB Table Shift: 4
5-3-5-4-5-5-3
min GPU 635mV

![2560](https://github.com/hanai3Bi/Switch-OC-Suite/assets/4215938/55d89996-81b7-4af7-8d1f-dd8c65ea5e01)

Max in Crash Bandicoot 1 docked: ~54 FPS

![2560_c](https://github.com/hanai3Bi/Switch-OC-Suite/assets/4215938/5bacf974-6752-4992-a3e9-5408c8a88ecb)

**Reply by halop** [reply]
most power efficient: (~1W less than 2560MHz)

[Mariko]
Samsung K4U6E3S4AA-MGCL
CUSTOM_ADJ_ALL
2400 MHz
VDD2: 1150, VDDQ: 600
DVB Table Shift: 1
4-4-5-4-5-5-4

![2400](https://github.com/hanai3Bi/Switch-OC-Suite/assets/4215938/43dc508a-450d-4298-926d-26aa3c9fcda7)

Max in Crash Bandicoot 1 docked: ~52 FPS
